components/pedigree/resources/src/main/resources/pedigree/view/graphicHelpers.js

```diff
@@ -315,6 +315,34 @@
         return newSet;
     };
 
+    /**
+     * Attempts to make resizableElement in the dialog modal fit on screen by adjusting the resizableElement height
+	 *
+	 * @param {String} dialogSelectorClass
+	 * @param {String} resizableElementId
+	 * @param {Number} minResizableElementHeight
+	 * @param {Number} maxResizableElementHeight
+     */
+    GraphicHelpers.adjustPreviewWindowHeight = function(dialogSelectorClass, resizableElementId, minResizableElementHeight, maxResizableElementHeight) {
+        var pedigreeDialogue = $$('.'+dialogSelectorClass)[0];
+		var resizableElement = $(resizableElementId);
+        if (!pedigreeDialogue || !resizableElement) {
+            return;
+        }
+        var screenHeight = window.innerHeight - 10;
```

Comment:
because it was here https://github.com/phenotips/phenotips/pull/2146/files#diff-3a30a4eb55105151589d0845319bc80cL214